portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -20,27 +20,39 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 import java.util.ArrayList;
-import java.util.Arrays;
 import com.google.gson.Gson;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-  ArrayList<String> messages = new ArrayList<String>( 
-    Arrays.asList("msg1", 
-                  "msg2", 
-                  "msg3")); 
+  class Comment {
+    String email, comment;
+    public Comment(String email, String comment){
+      this.email=email;
+      this.comment=comment;
+    }
+  }
+  ArrayList<Comment> comments = new ArrayList<Comment>();
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    String messages_json = convertToJsonUsingGson(messages);
+    String comments_json = convertToJsonUsingGson(comments);
     response.setContentType("application/json;");
-    response.getWriter().println(messages_json);
+    response.getWriter().println(comments_json);
   }
   
-  private String convertToJsonUsingGson(ArrayList<String> arr) {
+  private String convertToJsonUsingGson(ArrayList<Comment> arr) {
```

Comment:
`private static` on this function (you're not using any member variables here, so it can be static)